### PR TITLE
Features/person details pl all

### PR DESCRIPTION
**Pull request by patrycja-i-lesniak**
I added a Person Details component. Please check its . I'm having trouble retrieving relevant data for person videos.
I did this big cleanup with the files.

### CODE REVIEW

**Review by KarolCiesluk — APPROVED** [review]
Try to fix rating placement in MovieTiles. Apart from that everything looks great! Bravo! 👏

**Comment by patrycja-i-lesniak** [comment]
I'm working on it right now, but I will add changes in a new branch. I had to change a bit throughout MovieTiles.

File: src/features/Tiles/MovieTiles/index.js [hunk 1]
```diff
@@ -12,55 +14,113 @@ import {
...
+                        <SiteTitle>
+                            {sectionName.charAt(0).toUpperCase() + sectionName.slice(1)}
+                        </SiteTitle>
+                        :
+                        <SiteTitle>
+                            {`Movie - ${sectionName} (${moviesList.length})`}
+                        </SiteTitle>
+                }
+                <List>
+                    {moviesList.map((movie, index) =>
```
Comment (WojciechBylica) [inline src/features/Tiles/MovieTiles/index.js:67]: please make destructuring:
 {moviesList.map(({
                        id,
                        poster_path,
                        title,
                        release_date,
                        character,
                        job,
                        genre_ids,
                        vote_average,
                        vote_count },
                        index) =>
                        
                        and remove all movies.[...] in map paths

File: src/features/Tiles/MovieTiles/styled.js [hunk 2]
```diff
@@ -138,3 +138,40 @@ export const Tag = styled.li`
         margin-bottom: 8px;
     }
 `;
+
+export const ShowMoreButton = styled.button`
```
Comment (WojciechBylica) [inline src/features/Tiles/MovieTiles/styled.js:142]: same thing: this styling is exactly the same as ShowMoreButton in PeopleTiles!

File: src/features/Tiles/PersonTile/styled.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    }
+
+    @media(max-width: ${({ theme }) => theme.breakpoints.xsmall}px) {
+        width: 114px;
+        height: 169px;
+        margin: 0 16px 0 0;
+    }
+`;
+
+export const Content = styled.div``;
```
Comment (WojciechBylica) [inline src/features/Tiles/PersonTile/styled.js:51]: no styling for Content... maybe it s enough to have just div in code?

File: src/features/Tiles/ShowMoreButton/index.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+                </ShowMore>
+            </>
+        ) : (
+            <>
+                {biography}
+            </> 
+        );
+};
+
+
```
Comment (WojciechBylica) [inline src/features/Tiles/ShowMoreButton/index.js:25]: empty line

**Review by WojciechBylica — CHANGES_REQUESTED** [review]
Pati, great job! all my suggestions are about code editing/cleaning. So I give request changes, only because of the fact, that this pull requests changes code in many places and I think It would be easier to edit it now.

File: src/features/PopularPeople/PersonDetails/styled.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
```
[the remark this answers is not in the document]
Comment (WojciechBylica) [reply src/features/PopularPeople/PersonDetails/styled.js:3]: and its almost the same as LoaderAndNoResultsWrapper...

File: src/features/PopularMovies/index.js [hunk 6]
```diff
@@ -11,7 +11,7 @@ import { useQueryParameter } from "../../useQueryParameter";
 import searchQueryParamName from "../../searchQueryParamName";
 import { selectPopularMoviesStatus, fetchPopularMoviesLoading } from "./popularMoviesSlice";
 
-const MovieList = () => {
+const PopularMovieList = () => {
```
[the remark this answers is not in the document]
Comment (patrycja-i-lesniak) [reply src/features/PopularMovies/index.js:14]: Yes, I corrected.

File: src/features/PopularPeople/PersonDetails/styled.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (patrycja-i-lesniak) [reply src/features/PopularPeople/PersonDetails/styled.js:3]: You're right, I moved  wrappers to common/Wrappers

File: src/features/Tiles/BigMovieTile/index.js [hunk 7]
```diff
@@ -33,7 +34,7 @@ export const BigMovieTile = () => {
 
     return (
         <MovieDetailsTile>
-            <Image src={poster} alt="" />
+                <Image src={movieData.poster_path ? poster : noMoviePhoto} alt={`${title} poster`} />
```
[the remark this answers is not in the document]
Comment (patrycja-i-lesniak) [reply src/features/Tiles/BigMovieTile/index.js:37]: Thx :)

File: src/features/Tiles/MovieTiles/index.js (hunk 1 above)
Comment (patrycja-i-lesniak) [reply src/features/Tiles/MovieTiles/index.js:67]: Done!

File: src/features/Tiles/PersonTile/styled.js (hunk 3 above)
Comment (patrycja-i-lesniak) [reply src/features/Tiles/PersonTile/styled.js:51]: Styles added in a subsequent revision

File: src/features/Tiles/ShowMoreButton/index.js (hunk 4 above)
Comment (patrycja-i-lesniak) [reply src/features/Tiles/ShowMoreButton/index.js:25]: Corrected.

**Review by WojciechBylica — APPROVED** [review]

**Pull request merged**